tools/props/Versions.props

```diff
@@ -27,8 +27,7 @@
   </PropertyGroup>
   <!-- NetFx and NetCore project dependencies -->
   <PropertyGroup>
-    <AzureIdentityVersion>1.11.3</AzureIdentityVersion>
-    <MicrosoftIdentityClientVersion>4.60.3</MicrosoftIdentityClientVersion>
+    <AzureIdentityVersion>1.11.4</AzureIdentityVersion>
```

Comment:
Thoughts about bumping this 1.12.0

Reply:
Actually no, not in a patch

Reply:
@ErikEJ what I do notice is that this is a new dependency from this batch version going forward k

Reply:
?? Sure, but not a new major version... (Or I do not understand what you mean)

Reply:
As it currently stands there is a dependency on Microsoft.Identity however with this pr we are removing that dependency & introducing a new dependency on Azure.Identity hence feel that we should be using the latest.